src/EditorFeatures/CSharpTest/Diagnostics/GenerateType/GenerateTypeTests_Dialog.cs

```diff
@@ -2006,114 +2006,6 @@ public interface DDD
 assertGenerateTypeDialogOptions: new GenerateTypeDialogOptions(true, TypeKindOptions.Interface, false));
         }
 
-        [WorkItem(861362)]
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateType)]
-        public void GenerateTypeInMemberAccessExpression()
```

Comment:
In general, there shouldn't be many cases where we delete a unit test. The scenarios are still important. So, it's often better to turn them into negative tests.